### PR TITLE
Remove PD/PCA attachment file type options from fixture

### PR DESCRIPTION
**Pull request by reinbach**
[ch10183]

There is a migration that cleans up these file types (PD/PCA), but In staging it appears the Ghana was added after this migration was run.

### CODE REVIEW

File: src/etools/applications/core/data/attachments_file_types.json [hunk 1]
```diff
@@ -169,16 +149,6 @@
             "code": "tpm_report"
         }
     },
-    {
```
Comment (domdinicola) [inline src/etools/applications/core/data/attachments_file_types.json]: do we need to remove also from tpm?

Comment (reinbach) [reply src/etools/applications/core/data/attachments_file_types.json]: I thought it needed to be removed from all? Is that not the case?

Comment (domdinicola) [reply src/etools/applications/core/data/attachments_file_types.json]: @ntrncic do we need `PCA` document type in TPM? Shall we check with the Business Owner?

Comment (ntrncic) [reply src/etools/applications/core/data/attachments_file_types.json]: no we don't need it. @reinbach @domdinicola

**Pull request merged**